apps/settings/js/modules/customized_network_type_map.js

```diff
@@ -0,0 +1,36 @@
+define(function(require) {
+  'use strict';
+
+  const KEY = 'operatorResources.data.icon';
+
+  var initialized = false;
+  var map = null;
```

Comment:
Can we use `{}` for map ? If one day we did get an error from mozSettings, we can still make the code work when using `CustomizedNetworkTypeMap`